packages/react/src/components/DataTable/TableToolbarSearch.js

```diff
@@ -52,6 +54,16 @@ const TableToolbarSearch = ({
     }
   }, [controlled, expandedState]);
 
+  useEffect(
+    () => {
+      if (defaultValue) {
+        onChangeProp('', defaultValue);
+      }
+    },
+    //eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
If the `defaultValue` changes would we want to update the input value or no? Just was curious if that was why the ignore is here.

Reply:
@asudoh and I went back and forth here. I originally had it updating if DefaultValue was updated. I don't really have a strong preference either way.

Reply:
Change in `defaultValue` in `<input>` doesn't update its value, so my suggestion is taking a cue from there.